src/jsx/components/input_info.jsx

```diff
@@ -63,6 +77,8 @@ class InputInfo extends React.Component {
                 <a href={window.location.href+"/original_file/original.fasta"}>Original file</a>
               </li>),(<li className="dropdown-item">
                 <a href={window.location.href+"/log.txt/"}>Analysis log</a>
+              </li>), (<li className="dropdown-item">
+                <a onClick={()=>this.open('msa')}>View MSA</a>
               </li>)] : null}
               {show_partition_button ? (<li className="dropdown-item">
```

Comment:
Looks like you were just following the format that was already there but the jsx inside the conditional seems like it could use some indenting/re-formatting to make it more readable.

Reply:
We shouldn't spend time toying around with formatting. Just run it through `prettier` before submitting a PR.

Reply:
As stated by prettier.io : 

> No need to discuss style in code review
> Saves you time and energy

Reply:
Sounds good to me! Do we have a standard prettier configuration that includes jsx?

Reply:
FYI - I just added prettier to the GUI repo and it seems to handle jsx pretty well out of the box.

Reply:
I've been happy with the defaults, personally. If there's something with the defaults that makes it difficult for one of us read, then we shouldn't hesitate to make it known.